### PR TITLE
More clarifications of logical vs physical pointer type for SPV_EXT_physical_storage_buffer

### PR DESCRIPTION
**Pull request by jeffbolznv**

### CODE REVIEW

**Comment by cmarcelo** [comment]
LGTM.

**Comment by johnkslang** [comment]
Looks good to me too.

File: extensions/EXT/SPV_EXT_physical_storage_buffer.asciidoc [hunk 1]
```diff
@@ -131,6 +152,29 @@ to:
...
+
+    *OpVariable* can allocate an object whose type is a pointer type, if the
+    <<Storage_Class, Storage Class>> of the *OpVariable* is one of the
+    following: ...
+
+to:
+
+    *OpVariable* can allocate an object whose type is a
+    <<LogicalPointerType,logical pointer type>>, if the
+    <<Storage_Class, Storage Class>> of the *OpVariable* is one of the
```
Comment (johnkslang) [inline extensions/EXT/SPV_EXT_physical_storage_buffer.asciidoc]: BTW, mark-up wise, this is 'Storage Class', as it's meant to be that operand of the instruction.  Adding operand helps:

...if the 'Storage Class' operand of the *OpVariable*...

That's what makes it unambiguous about levels of indirection involved.

Comment (johnkslang) [reply extensions/EXT/SPV_EXT_physical_storage_buffer.asciidoc]: (I think the extension spec. is okay; the link doesn't work anyway.)

**Pull request merged**